specification/powerbiprivatelinks/resource-manager/Microsoft.PowerBI/stable/2020-06-01/powerbiprivatelinks.json

```diff
@@ -0,0 +1,1627 @@
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Specifies the id of the resource."
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Specifies the name of the resource."
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Specifies the type of the resource."
+        },
+        "location": {
+          "type": "string",
+          "description": "Specifies the location of the resource."
+        },
+        "etag": {
+          "type": "string",
+          "description": "Specifies the etag of the resource."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/PrivateEndpointConnectionProxy",
+          "description": "Specifies the private endpoint proxy resource."
+        }
+      }
+    },
+    "PrivateEndpointConnectionProxy": {
+      "title": "PrivateEndpointConnectionProxy",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Specifies the id of the resource."
+        },
+        "name": {
+          "type": "string",
+          "description": "Specifies the name of the resource."
+        },
+        "type": {
+          "type": "string",
+          "description": "Specifies the type of the resource."
+        },
+        "location": {
+          "type": "string",
+          "description": "Specifies the location of the resource."
+        },
+        "etag": {
+          "type": "string",
+          "description": "Specifies the etag of the resource."
+        },
+        "remotePrivateEndpoint": {
+          "$ref": "#/definitions/RemotePrivateEndpoint",
+          "description": "Specifies the remote private endpoint of the resource."
+        }
+      }
+    },
+    "PrivateEndpointConnection": {
+      "title": "PrivateEndpointConnection",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Specifies the id of the resource."
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Specifies the name of the resource."
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Specifies the type of the resource."
+        },
+        "systemData": {
+          "$ref": "#/definitions/SystemData",
+          "readOnly": true
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/PrivateEndpointConnectionProperties",
+          "description": "Specifies the properties of the private endpoint connection."
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "PrivateEndpointConnectionListResult": {
```

Comment:
>PrivateEndpointConnectionListResult [](http://example.com/codeflow?start=5&length=35)

Consider $ref of types from privatelinks.json and/or types.json below common-types.

Reply:
will keep the current property.